### PR TITLE
Article/the scourge of silos

### PR DESCRIPTION
**Pull request by Hamo69**
Completed rant about what DevOps is and the effect of silos. Not sure it hits the mark and I may have rambled somewhat. Needs a thorough review and comments.

### CODE REVIEW

File: _posts/2017-09-01-the-scourge-of-silos.markdown [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title:  "The Scourge of Silos"
+date:   2017-09-01 04:48:58 +0000
+categories: bdd tdd devops pairing mobbing
+author: Hamish Tedeschi
+image: img/silos1.png
```
Comment (temyers) [inline _posts/2017-09-01-the-scourge-of-silos.markdown]: Should we create sub-folders for the images?  Or not there yet?

Comment (Hamo69) [reply _posts/2017-09-01-the-scourge-of-silos.markdown]: I had that thought as I was adding the images. I'm happy to re-jig

Comment (Hamo69) [reply _posts/2017-09-01-the-scourge-of-silos.markdown]: Any content related comments?

**Comment by Hamo69** [comment]
@temyers @nickj69 can I merge?

File: _posts/2017-09-01-the-scourge-of-silos.markdown [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+---
+This is one of those articles that may stir up a bit of angst with people, because essentially it is an opinion piece. Not driven by data, merely driven by what I think works best. It was provoked by a brief conversation on social media and got me thinking about why I think the way I do. The conversation was based on the following post by [Josh Partogi](https://www.linkedin.com/in/jpartogi/), which was on the money:
```
Comment (temyers) [inline _posts/2017-09-01-the-scourge-of-silos.markdown]: Ok so this first paragraph is good conversational style to get started.  However, looking at the SEO crap from WP, it mentioned that the first sentence should sum up the article to bump up the ratings.

File: _posts/2017-09-01-the-scourge-of-silos.markdown [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+So why do organisations create a DevOps Team, which sits in its own bubble and has its own reporting structure, which essentially look after infrastructure, sometimes as code, in my experience? I think a lot of it comes down to organisational structure and command and control leadership styles. 
+
+![Command and Control Management Picture](/img/commdandcontrol.png)
+
+Command and control assumes a leader knows best and that they know where the organisation is going (goals, outcomes) and have a plan for how to get there (process). With a term as fuzzy as "DevOps", as illustrated by Josh's post, should we expect IT leaders in enterprise organisations, who have gone from Waterfall to Agile and now DevOps to be in touch enough to understand the nuance of it? Not in my experience. 
+
+So enter matrix management, which was designed to solve all these problems, by creating product centric delivery teams, but still retaining line management.
```
Comment (temyers) [inline _posts/2017-09-01-the-scourge-of-silos.markdown]: So here's the crux of your argument.

I like it, but it's currently buried on line 43.

You set the scene well and it's eloquently written - but it's too long I think.

Also the picture doesn't describe matrix management - cos it doesn't indicate the line management chain off to the side

File: _posts/2017-09-01-the-scourge-of-silos.markdown [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+![Matrix Management Picture](/img/matrix-man.png)
+
+The problem with matrix manangement is that fosters all sorts of problems. Political bullshit, being one. In this scenario, a team of individuals from different parts of the organisation are brought together to deliver something. In theory, this is great, but the problem is that line management for these indivduals sits off to one side. Any successes will be attributed to the members of that line manager and inversely any failures will be as a result of the other team members not aligning with that manager. The biggest problem, as a result of this, is the time to make decisions. It effectively widens the feedback loop (again) by involving interested third parties and will disenfranchise the team actually doing the work.
+
+The modern equivalent of matrix management in IT teams is the Spotify Model. Not that is actually a model. But expensive consultancies are paid loads to roll out a "model" which effectively propogates this shit. Guilds and tribes may have been formed with good intentions, but are just abused by the middle management that become the issue.
```
Comment (temyers) [inline _posts/2017-09-01-the-scourge-of-silos.markdown]: 🤔

Comment (temyers) [reply _posts/2017-09-01-the-scourge-of-silos.markdown]: Do you want to mention "there is no Spotify model" :)

File: _posts/2017-09-01-the-scourge-of-silos.markdown (hunk 2 above)
Comment (temyers) [reply _posts/2017-09-01-the-scourge-of-silos.markdown]: Also, the link here is just to his profile, not the post - can you link to the post itself?  

Or is the quote the whole post? - confusing for me

**Comment by temyers** [comment]
Ok I've approved, since my comments are more style and personal preference - if you think it's fine I won't stop the review

File: _posts/2017-09-01-the-scourge-of-silos.markdown (hunk 4 above)
Comment (Hamo69) [reply _posts/2017-09-01-the-scourge-of-silos.markdown]: why chin scratch??

**Pull request merged**